src/main/java/com/faforever/client/preferences/GeneralPrefs.java

```diff
@@ -9,6 +9,7 @@
 public class GeneralPrefs {
 
   BooleanProperty disableSteamStart = new SimpleBooleanProperty(false);
+  BooleanProperty showCyrillicWarning = new SimpleBooleanProperty(true);
```

Comment:
Should default to false
```suggestion
  BooleanProperty showCyrillicWarning = new SimpleBooleanProperty(false);
```

Reply:
False? Then the warning will not be displayed at all even once. Maybe you mean that I should to rename from `showCyrillicWarning ` to `ignoreCyrillicWarning` variable?  Then it would make sense

Reply:
Ah yes sorry I read this wrong, you are correct. Either way is fine